src/wmtk/simplex/link_condition.cpp

```diff
@@ -141,17 +142,15 @@ bool link_condition(const TetMesh& mesh, const Tuple& edge)
                         input_v)) {
                     // tuple edge is _v - x
                     ret.push_back(mesh.switch_edge(mesh.switch_vertex(_f.tuple())));
+                } else if ((utils::SimplexComparisons::equal(
```

Comment:
As code coverage indicates, I am pretty sure that `cofaces_single_dimension` already gives you the tuples with the desired orientation.